modules/web/web-vhost/src/test/java/com/enonic/xp/web/vhost/impl/VirtualHostInternalHelperTest.java

```diff
@@ -13,6 +13,19 @@
 public class VirtualHostInternalHelperTest
 {
 
+    @Test
+    public void testNormalizePath()
+    {
+        final HttpServletRequest req = mock( HttpServletRequest.class );
+        when( req.getRequestURI() ).thenReturn( "/a/other/service" );
+
+        assertEquals( "/", VirtualHostInternalHelper.normalizePath( null ) );
```

Comment:
verify if null safety is needed

Reply:
if `source` or `target` field is not set, then null is coming and we handle it like `/`

Reply:
But this work should be done in VirtualHostConfigMap, in order for Vhost API to work correctly